fiftyone/core/dataset.py

```
@@ -1100,6 +1100,321 @@ def default_skeleton(self, skeleton):
+                    cx=640.0, cy=360.0,
+                ),
+            }
+
+            # Edit existing intrinsics
+            dataset.camera_intrinsics["camera_front"].k1 = -0.12
+            dataset.save()  # must save after edits
+        """
+        return self._doc.camera_intrinsics
+
+    @camera_intrinsics.setter
+    def camera_intrinsics(self, intrinsics):
```

Ruff (0.14.10)

Missing return type annotation for private function `camera_intrinsics`

Add return type annotation: `None`

(ANN202)